### ISSUE TITLE
Code Coverage Reporting

### ISSUE THREAD

**Issue by jgodi** [jgodi/angular2-fullstack-starter#3]
Add the ability to report the code coverage and make the code coverage percentage go off the ES6/TS code if possible.

**Comment by xiehan** [comment]
If you get this working, could you give me a ping? I've got the same setup as you (Angular2 + Webpack + ES6 but not TS + Babel) and have not been able to get it working, nor have I found anyone else with this setup who's been able to get it to work.

**Comment by jgodi** [comment]
Yes. I have it working for the webpack version. I will make sure this is updated and shows it.

**Comment by jgodi** [comment]
I dont have the code anymore (we swapped from using webpack). But I just used the https://www.npmjs.com/package/istanbul-instrumenter-loader, inside the karma.conf.
